src/calculus.jl

```diff
@@ -0,0 +1,10 @@
+
+
+function diff(b1::Basic, b2::Basic)
```

Comment:
Probably. I'm trying to see how to create the classes you want from just
that. If I figure it out, I'll let you know.

On Fri, Mar 25, 2016 at 4:09 PM, Isuru Fernando <notifications@github.com>
wrote:

> In src/calculus.jl
> <https://github.com/symengine/SymEngine.jl/pull/17#discussion_r57485020>:
>
> > @@ -0,0 +1,10 @@
> > +
> > +
> > +function diff(b1::Basic, b2::Basic)
>
> I can add a method to get the enum from the class name as a string. TypeID
> basic_get_class_id(char *). That would be enough right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/SymEngine.jl/pull/17/files/7343bd913def49c83b58a274d6363243ba1b9b57#r57485020>
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu